pkg/api/openapi/minder/v1/minder.swagger.json

```diff
@@ -1418,6 +1418,50 @@
         "tags": [
           "ProvidersService"
         ]
+      },
+      "patch": {
+        "operationId": "ProvidersService_PatchProvider",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/v1PatchProviderResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "patch",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/v1ProviderPatch"
```

Comment:
This feels a little odd in comparison with `v1CreateProviderRequest`, which has fields:

* `context`
* `provider`
  * `name`
  * `class`
  * `project`
  * `version`
  * `implements`
  * `config`
  * `authFlows`
  * `parameters`
  * `credentialsState`

It feels like you might want to make the body of the message be a `Provider`, and then disallow updates to anything other than the `config` field at the moment.

Reply:
thanks, I didn't want to expose all the provider attributes at first, but you're right that if we don't make use of them then it doesn't matter and forward compatibility is better.